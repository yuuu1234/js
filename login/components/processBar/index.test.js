import React from 'react';
import renderer from 'react-test-renderer';

import ProgressBar from './';

test('ProgressBar screen renders correctly', () => {
  const tree = renderer.create(<ProgressBar index={0} />).toJSON();
  expect(tree).toMatchSnapshot();
});
