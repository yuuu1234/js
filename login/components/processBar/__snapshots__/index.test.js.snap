// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ProgressBar screen renders correctly 1`] = `
<View
  style={
    Object {
      "height": 80,
    }
  }
>
  <SvgMock
    width={750}
  />
</View>
`;
